src/website/src/pages/ec/components/navigation/breadcrumb/demo/index.js

```diff
@@ -0,0 +1,21 @@
+import dataSimple from '@ecl/specs-component-breadcrumb/demo/data--simple';
+import dataLong from '@ecl/specs-component-breadcrumb/demo/data';
+import templateCore from '@ecl/twig-component-breadcrumb-core/breadcrumb-core.html.twig';
+import templateStandardised from '@ecl/twig-component-breadcrumb-standardised/breadcrumb-standardised.html.twig';
+import templateHarmonised from '@ecl/twig-component-breadcrumb-harmonised/breadcrumb-harmonised.html.twig';
+import { correctSvgPath } from '@ecl/website-utils';
+
+export const simpleContentCore = templateCore(correctSvgPath(dataSimple));
```

Comment:
it's just variable names, but this is the rendered component, kind of confusing naming it `simpleContentCore`, it makes you think it's stil about the data

Reply:
I'd add a suggestion here for the following import

```js
import templateHarmonised from '@ecl/twig-component-breadcrumb-harmonised/breadcrumb-harmonised.html.twig';
```

Whenever possible, please name the import as `template`. Also a naming-related request, though it will really help down the line when some day we decide to change demo rendering on the website. Searching for `import template from` will be a working pattern for uniform refactoring.

Reply:
@planctus Thks, changed variables naming `simpleContentCore` to `breadcrumbSimpleCore`

Reply:
@kalinchernev I split the demo file into 3 family files `core`, `standardised` and `harmonised`